data/card_catalogue.json

```diff
@@ -1257,18 +1209,28 @@
         },
         {
           "title": "Deflect",
-          "upgrade": "",
+          "upgrade": "Golan's Idol",
           "tier": "red",
           "details": {
-            "rules": "Any damage dealt to you until your next turn is instead dealt to your attacker",
+            "rules": "Any damage dealt to you from the next attack targeting you is instead dealt to your attacker",
```

Comment:
Downshifting this effect to red. It's incredibly good in combat but not story shattering